addons/website/views/website_templates.xml

```diff
@@ -57,6 +57,8 @@
         <script type="text/javascript" src="/website/static/src/js/content/zoomodoo.js"/>
 
         <script type="text/javascript" src="/website/static/src/js/menu/navbar.js"/>
+
+        <script type="text/javascript" src="/website/static/src/js/html_editor.test.js"/>
```

Comment:
There is a "tour" folder :)

Reply:
Yup, I placed it there in the first place but it is only in website module that we got a `tour` folder.
Almost everywhere else the tour-test are placed in js root folder.
It is not really a tour but a test (but not a QUnit test in `tests`) so idk, you definitely want it in the tour folder? In website_sale eg ever tour-test are in js root folder.

Reply:
There is already a tour test in that folder ;) But I suppose we can move all the tests in a proper tests folder when we will do task https://www.odoo.com/web#id=1934445&action=327&model=project.task&view_type=form&menu_id=4720, so you can forget about this.